packages/ui/src/components/shared/export/QrCode.tsx

```diff
@@ -6,15 +6,18 @@ import { base64Encode } from '@polkadot/util-crypto';
 import { AccountBackup, QrBackup } from '@coong/keyring/types';
 import { Download } from '@mui/icons-material';
 import { Button } from '@mui/material';
+import CoongLogo from 'assets/images/coong-logo.png';
 import FileSaver from 'file-saver';
 import useQrCodeSize from 'hooks/useQrCodeSize';
-import { TransferableObject, Props } from 'types';
+import { Props, TransferableObject } from 'types';
 
 interface QrCodeProps extends Props {
   value: QrBackup;
   object: TransferableObject;
 }
 
+const defaultSize = 500;
```

Comment:
For global constants, we should use this format:
```suggestion
const DEFAULT_SIZE = 500;
```